labman/gui/handlers/process_handlers/test/test_gdna_compression_process.py

```diff
@@ -35,7 +35,7 @@ def test_get_gdna_plate_compression_process_handler(self):
         self.assertEqual(response.code, 404)
 
     def test_post_gdna_plate_compression_process_handler(self):
-        data = {'plates': json_encode(['22', '22']),
+        data = {'plates': json_encode(['22', '28']),
```

Comment:
Why the change from 22 to 28?

Reply:
The original test was "compressing" the SAME plate (plate 22) twice onto a larger plate.  According to Greg, this is not possible, which makes it a bad choice for a test case. Now that we have expanded the test database to actually *have* more than one gdna plate, the test has been changed to reflect the realistic situation of compressing two different gdna plates onto a larger plate rather than the impossible situation of compressing the same plate twice.

Reply:
thanks